keps/sig-node/4381-dra-structured-parameters/README.md

```diff
@@ -546,6 +548,12 @@ the DRA drivers providing content for those objects. It might be possible to
 support version skew (= keeping kubelet at an older version than the control
 plane and the DRA drivers) in the future, but currently this is out of scope.
 
+For network-attached resources, the DRA driver is responsible for discovering
```

Comment:
Currently, ResourceSlice is generated with the name `<node_name>-<driver_name>-<random_string>`, but if setting the NodeSelector of ResourceSlice, would it be `<driver_name>-<random_string>`?
https://github.com/kubernetes/kubernetes/blob/v1.30.0/pkg/kubelet/cm/dra/plugin/noderesources.go#L470

Reply:
How to name those ResourceSlices would be entirely up to the driver. What matters isn't the name, only the content.

I am moving the ResourceSlice controller out of kubelet into the `k8s.io/dynamic-resource-allocation` package as part of https://github.com/kubernetes/kubernetes/pull/124274, so drivers could reuse that (eventually - right now in that PR it doesn't support network-attached resources yet).